ow_system_plugins/base/bol/avatar_service.php

```diff
@@ -1023,6 +1023,13 @@ public function createAvatar( $userId, $isModerable = true, $trackAction = true)
             return false;
         }
 
+        if ( !getimagesize($path) )

```

Comment:
merge this logic with the method above like:
if (  !UTIL_File::validateImage($path) || !getimagesize($path) )